src/ziggurat/resource/dead_set.clj

```diff
@@ -30,21 +30,21 @@
   (some? channel))
 
 (defn retry-enabled? []
-  (get-in-config [:retry :enabled]))
+  (get-in (retry-config) [:enabled]))
```

Comment:
since you are using `retry-config` function only once here. I don't think this function makes sense. You can use `(get-in-config [:retry])` instead.

Reply:
Sure, since it is straightforward, change it to use `get-in-config`